bot/controllers/games.js

```diff
@@ -106,12 +105,8 @@ class GamesController extends BaseController {
       },
     };
     fetch('https://icanhazdadjoke.com/', options)
-      .then((res) => {
-        res.text();
-      })
-      .then((body) => {
-        return `Here is your joke ${message.author.username}!! ${body}`;
-      });
+      .then(res => res.text())
+      .then(body => message.reply(`Here is your joke ${message.author.username}!! ${body}`));
   }
 }
 
```

Comment:
Possible solution to this would be to use async / await for the fetch, that way the reply waits for the joke to return before posting.

Reply:
the function is already an async function